testnets/xarchainceriestestnet/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "xarchainceriestestnet",
+  "assets": [
+    {
+      "description": "Stake is the native cryptocurrency of the Arcana app chain (ceries), designed to orchestrate bridge-less interoperability between multiple blockchains",
+      "extended_description": "Stake powers the Arcana app chain by supporting network consensus through staking, ensuring security and trustless coordination across chains.",
+      "denom_units": [
+        {
+          "denom": "stake",
+          "exponent": 0
+        }
+      ],
+      "base": "stake",
+      "name": "Xarchain ceries staking token",
+      "display": "stake",
+      "symbol": "stake",
```

Comment:
Please make symbol all uppercase (any lowercase component must reasonably derive from it's relationship to another asset, which doesn't apply here)